charts/tornjak/values.yaml

```diff
@@ -113,7 +113,7 @@ tornjak:
       socketFile: api.sock
       # ingress - if Ingress required, don't use 'http://' prefix
       # keep empty if local, or direct access available
-      ingress: # 👈 Use it with Cloud Ingress for Backend
+      ingress:  # 👈 Use it with Cloud Ingress for Backend
```

Comment:
should this also have double quote like in line 101?

Reply:
This should work and I wanted to demonstrate it works for both cases, but you are right, we should be consistent

Reply:
This should work and I wanted to demonstrate it works for both cases, but you are right, we should be consistent